charts/scand-manager/values.yaml

```diff
@@ -0,0 +1,140 @@
+## Image for scand-manager Pod
+#
+image:
+  repository: 42crunch/scand-manager
+  pullPolicy: IfNotPresent
+  tag: v1.2.2
+
+## Parameters for on-premises conformance scan Job
+## ref: https://docs.42crunch.com/latest/content/tasks/scan_api_conformance.htm
+#
+scanJob:
+  platformService: services.us.42crunch.cloud:8001
+  scandImage: 42crunch/scand-agent:latest
+  expirationTime: 86400
+  ## PodConfig for conformance scan Job
+  ## Currently only support Pod Affinity rule
+  ## ref: https://github.com/42Crunch/scand-manager/tree/master#optionally-configure-pod-affinity-rules
+  #
+  podConfig:
+    enabled: false
+    ## Affinity for conformance scan Job's Pod assignment
+    ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+    ## e.g.:
+    ## affinity:
+    ##   nodeAffinity:
+    ##     requiredDuringSchedulingIgnoredDuringExecution:
+    ##       nodeSelectorTerms:
+    ##       - matchExpressions:
+    ##         - key: kubernetes.io/os
+    ##           operator: In
+    ##           values:
+    ##           - linux
+    #
+    affinity: {}
+
+## Firewall injection for scand-manager Pod
+## firewall-k8s-injector must be deployed to take effect
+## ref: https://github.com/42Crunch/firewall-k8s-injector
+#
+firewallInjection:
+  enabled: false
+  protectionToken: "apifirewall-protection-token"
+  containerPort: 8090
+  envConfigmap: "apifirewall-props"
+  targetUrl: http://localhost:8090
+  serverName: scand-manager.local
+
+## RBAC for scand-manager Pod
+## ref: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+#
+rbac:
+  enabled: true
+  serviceAccount:
+    annotations: {}
+    nameOverride: ""
+  rules:
+  - apiGroups: ["batch", "extensions"]
+    resources: ["jobs"]
+    verbs: ["get", "create", "delete", "list"]
+  - apiGroups: [""]
+    resources: ["pods"]
+    verbs: ["get", "list"]
+  - apiGroups: [""]
+    resources: ["pods/log"]
+    verbs: ["get"]
+
+## Service for scand-manager
+## ref: https://kubernetes.io/docs/concepts/services-networking/service/
+#
+service:
+  type: ClusterIP
+  port: 8090
+
+## Ingress for scand-manager Service
+## ref: https://kubernetes.io/docs/user-guide/ingress/
```

Comment:
Docs have moved to https://kubernetes.io/docs/concepts/services-networking/ingress/